### PR TITLE
Telegraf as a service 

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/2386

### CODE REVIEW

File: content/telegraf/v1.18/administration/windows_service.md [hunk 1]
```diff
@@ -8,37 +8,43 @@ menu:
...
 https://docs.microsoft.com/en-us/powershell/scripting/windows-powershell/starting-windows-powershell?view=powershell-7#with-administrative-privileges-run-as-administrator).
 {{% /note %}}
 
-1. Download the Telegraf binary and unzip its contents to `C:\Program Files\InfluxData\Telegraf`.
+## Before you begin
+
+The Telegraf installation includes an executable and configuration file. To simplify upgrades, consider the following:
+
+- **Setting up Windows symbolic link (Symlinks) to point to the Telegraf executable**. This option lets you easily roll back your upgrades, and maintain your custom configuration.
+- **Whether to use one or multiple Telegraf configurations**. You may want multiple configs if...
```
Comment (sjwang90) [inline content/telegraf/v1.18/administration/windows_service.md:26]: something after the `...`?

**Comment by sjwang90** [comment]
@xe-nvdk if you want to take a look.

Also this just is fixes for #2386 correct? The "Running multiple Telegraf instances as a service in Windows" from #2116 still needs to be added.

File: content/telegraf/v1.18/administration/windows_service.md (hunk 1 above)
Comment (sjwang90) [reply content/telegraf/v1.18/administration/windows_service.md:26]: @kelseiv Should this be using "multiple Telegraf instances" to possibly direct users to information about running multiple Telegraf instances

Comment (kelseiv) [reply content/telegraf/v1.18/administration/windows_service.md:26]: Thanks, @sjwang90! @pierwill plans to touch base with you on this one. Basically, in what use cases would users want to use multiple configs vs 1?

**Comment by kelseiv** [comment]
> @xe-nvdk if you want to take a look.
> 
> Also this just is fixes for #2386 correct? The "Running multiple Telegraf instances as a service in Windows" from #2116 still needs to be added.

Yes, thank you for verifying, @sjwang90, @pierwill plans to take a look at this one too.

File: content/telegraf/v1.18/administration/windows_service.md (hunk 1 above)
Comment (kelseiv) [reply content/telegraf/v1.18/administration/windows_service.md:26]: And the second part...prob makes sense to add "multiple Telegraf instances" to the list of things to consider? To distinguish between when folks want to use multiple instances and when to use multiple configs?

**Review by pierwill — COMMENTED** [review]
pong

**Review by pierwill — APPROVED** [review]

File: content/telegraf/v1.18/administration/windows_service.md [hunk 2]
```diff
@@ -10,35 +10,60 @@ menu:
...
+  You may want multiple configs if...
+-->
+
+## Download and run Telegraf as a Windows service
 
 {{% note %}}
 Installing a Windows service requires administrative permissions.
-Be sure to [launch Powershell as administrator](
-https://docs.microsoft.com/en-us/powershell/scripting/windows-powershell/starting-windows-powershell?view=powershell-7#with-administrative-privileges-run-as-administrator).
+Make sure you a running PowerSheel as an administrator:
```
Comment (kelseiv) [inline content/telegraf/v1.18/administration/windows_service.md:29]:
```suggestion
To run PowerShell as an administrator, 
```

File: content/telegraf/v1.18/administration/windows_service.md [hunk 3]
```diff
@@ -10,35 +10,60 @@ menu:
...
 {{% /note %}}
 
-1. Download the Telegraf binary and unzip its contents to `C:\Program Files\InfluxData\Telegraf`.
-2. In PowerShell, run the following as an administrator:
+In PowerShell _as an administrator_, do the following:
+
+1. Download `telegraf-{{< latest-patch >}}_windows_amd64.zip` and extract its contents
+   to `C:\Program Files\InfluxData\telegraf\`:
+   ```powershell
+   > wget https://dl.influxdata.com/telegraf/releases/telegraf-{{< latest-patch >}}_windows_amd64.zip -UseBasicParsing -OutFile telegraf-{{< latest-patch >}}_windows_amd64.zip
```
Comment (kelseiv) [inline content/telegraf/v1.18/administration/windows_service.md:38]: Is double `telegraf-{{< latest-patch >}}_windows_amd64.zip` needed?

**Pull request merged**